var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -137,6 +143,8 @@ def configure_args(self):
             make.add_default_arg("CXXFLAGS=%s" % self.compiler.cxx11_flag)
             args.append("CXXFLAGS=%s" % self.compiler.cxx11_flag)
 
+        config_args.extend(self.enable_or_disable("single-mpi-lib"))
```

Comment:
Doh: There is no variable `config_args`.

`args` is the correct variable. Please test that it works...:
```suggestion
        args.extend(self.enable_or_disable("single-mpi-lib"))
```